src/interface_adapter/settings/SettingsController.java

```diff
@@ -0,0 +1,19 @@
+package interface_adapter.settings;
+
+import entity.User;
+import use_case.settings.SettingsInputBoundary;
+import use_case.settings.SettingsInputData;
+
+public class SettingsController {
+
+  final SettingsInputBoundary settingsUseCaseInteractor;
+
+  public SettingsController(SettingsInputBoundary settingsUseCaseInteractor) {
+    this.settingsUseCaseInteractor = settingsUseCaseInteractor;
+  }
+
+  public void execute(User user) {
```

Comment:
Similarly with this comment

Reply:
Actually, to properly do the CA design process, interfaces should be fully fleshed out first